.github/workflows/auto-release.yml

```diff
@@ -44,14 +44,34 @@ jobs:
           path: local_kernel
           fetch-depth: 0
 
-      - name: Release
+      - name: Configure git identity
         run: |
-          # Configure repo for push
           git config --global user.name ${{ github.actor }}
           git config --global user.email ${{ github.actor }}@users.noreply.github.com
+
+      - name: create a new branch that references commit id
+        working-directory: ./local_kernel
+        run: git checkout -b ${{ github.event.inputs.version_number }} ${{ github.event.inputs.commit_id }}
+
+      - name: Generate SBOM
+        uses: xlin7799/CI-CD-Github-Actions/sbom-generator@generic
```

Comment:
This should refer to the FreeRTOS Github org.

Reply:
Thanks for catching this!